### ISSUE TITLE
Establish Git-Unity interaction

### ISSUE THREAD

**Issue by Dstew4444** [Evan-HR/Handits#7]
-Someone create Unity project
-All members should access project and open on Unity on local machines
-3 points
-Ming to push initial project

**Comment by Evan-HR** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Issue #7: Establish Git-Unity Interaction](https://trello.com/c/OtSfIKA0/27-issue-7-establish-git-unity-interaction)

**Comment by Dstew4444** [comment]
Steps to follow to sync Github and Unity:
1) First create a forked version on your Github account
https://help.github.com/articles/fork-a-repo/

2) Sync your fork of the repo with upstream repo(original repo)
https://help.github.com/articles/syncing-a-fork/

**Comment by Evan-HR** [comment]
Danny Successfully opened the file Ming posted, there was a slight complication of different versions of Unity, but it worked out fine. Let's all try to update Unity to the newest available version

**Comment by Dstew4444** [comment]
I have reopened this issue because I think we should all be able to access and manage the project without any underlying issues. As of right now, when I attempt to open Ming's example project, I get an error indicating that our two versions of Unity aren't compatible. I have recently updated to the newest version of Unity (2018.2.11f1).
<img width="420" alt="screen shot 2018-10-05 at 1 02 59 pm" src="https://user-images.githubusercontent.com/23229504/46549370-c2e5de80-c89f-11e8-8431-c9a2cdda0f9a.png">
